bench/bench.js

```diff
@@ -10,7 +10,7 @@ function benchmark(name, setup, action) {
 
   // Warm up the JIT.
   var start = Date.now();
-  while ((Date.now() - start) < 10000 /* 10 seconds */) {
+  while ((Date.now() - start) < 5000 /* 5 seconds */) {
```

Comment:
Unfortunately, because each parse takes so long, we can't just run it n times before measuring, and I don't think there is a better way.